libraries/AP_HAL_ChibiOS/hwdef/TBS_LUCID_PRO/README.md

```diff
@@ -0,0 +1,99 @@
+# TBS LUCID PRO Flight Controller
+
+The TBS LUCID PRO/Freestyle is a flight controller produced by [Team Black Sheep](https://www.team-blacksheep.com/).
+
+## Features
+
+ - MCU - STM32F405 32-bit processor running at 168 MHz
+ - IMU - ICM42688 (Freestyle) or MPU6000 (Pro)
+ - Barometer - SPL06
+ - OSD - AT7456E
+ - Onboard Flash: 4MByte
+ - 6x UARTs
+ - 6x PWM Outputs (4 Motor Output, 1 Camera PWM and 1 LED)
+ - Battery input voltage: 2S-6S
+ - BEC 3.3V 0.5A
+ - BEC 5V 3A
+ - BEC 9V 3A for video
+
+## Pinout
+
+![TBS LUCID PRO/Freestyle Board](TopBottom.png "TBS LUCID PRO/Freestyle")
+
+## UART Mapping
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+ - SERIAL0 -> USB
+ - SERIAL1 -> UART1 (RX, DMA-enabled)
+ - SERIAL2 -> UART2 (ESC Telem)
+ - SERIAL3 -> UART3 (HD RX, DMA-Enabled)
+ - SERIAL4 -> UART4 (SmartAudio)
+ - SERIAL5 -> UART5 (SBUS RX)
```

Comment:
```suggestion
 - SERIAL5 -> UART5 (User, RX5 is SBUS input and is inverted from pin)
```
it better be inverted from pin to UART5 RX or sbus wont work!

Reply:
Thanks, fixed